homeassistant/components/victron_gx_mqtt/strings.json

```
@@ -0,0 +1,1336 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "Already configured"
+    },
+    "error": {
+      "cannot_connect": "Failed to connect to host",
+      "unknown": "Unknown failure"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "operation_mode": "Operation mode",
+          "password": "[%key:common::config_flow::data::password%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "root_topic_prefix": "Optional root topic prefix",
+          "ssl": "[%key:common::config_flow::data::ssl%]",
+          "update_frequency": "Update frequency (seconds)",
+          "username": "[%key:common::config_flow::data::username%]"
+        },
+        "data_description": {
+          "host": "Hostname or IP address of Victron device, usually mDNS name like 'venus.local'",
+          "operation_mode": "Operation mode controls which Home Assistant entity types are created. 'read_only' exposes only sensors and binary_sensors (no writable entities), 'full' exposes all entity types (sensors, binary_sensors, numbers, selects, switches), 'experimental' is reserved for future use (behaves like 'full' today).",
+          "password": "Password for the Victron device, default is empty. This is not your VRM password.",
+          "port": "The MQTT port on the host. Normally it is 1883.",
+          "root_topic_prefix": "Root topic prefix if used via MQTT gateway.",
+          "ssl": "Indicates whether to use SSL to connect to the Victron device. Normally it is disabled.",
+          "update_frequency": "Update frequency in seconds. Default is 30 seconds. Set to 0 to update always.",
+          "username": "Username for the MQTT server, default is empty. Not needed by Victron devices. This is only needed if you route your MQTT messages through a non Victron server and it does require a username."
```

"Non" as a prefix usually merges with the word that follows, here with a name we need to use a hyphen:

```suggestion
          "username": "Username for the MQTT server, default is empty. Not needed by Victron devices. This is only needed if you route your MQTT messages through a non-Victron server and it does require a username."
```